romanesco/executors/pyspark.py

```diff
@@ -0,0 +1,37 @@
+import imp
+import json
+import sys
+from romanesco import spark
+
+def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
+    custom = imp.new_module("custom")
+
+    # TODO
+    # Ensure that the spark modules have been added to our path
+    # Create SparkContext - Should we share one across a workflow?
+    # Take any inputs and convert them to RDDS?
+    print 'Running pyspark'
+
+    sc = spark.create_spark_context()
```

Comment:
Yep, I figured for the moment as we are allowing users to run arbitrary python code on the server it would be pretty hard to protect this config and in many case the user is going to setup this config.